scripts/check-log-query-data.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+regex="log\.(Error|Debug|Info|Warn|Panic|Fatal)f*\(.*(query.Data).*\)"
```

Comment:
Note: this will not catch cases where the log text is long and query.Data is on a second line to comply with style line length. This will nevertheless limit the risk of logging sensitive query data.

Reply:
Good catch. Given that this is not urgent, I'll address it separately